src/main/java/bio/terra/cli/command/workspace/Update.java

```diff
@@ -29,11 +30,13 @@
     private String description;
   }
 
+  @CommandLine.Mixin WorkspaceOverride workspaceOption;
```

Comment:
I slightly prefer having the `--format=json` flag always at the end, since it's not related to the business logic. The command options should show up [alphabetically](https://picocli.info/#_unsorted_option_list) in the console for the user though.